scripts.py

```diff
@@ -202,15 +204,12 @@ def process_batch(self, batch):
 class MetadataCalculationScript(Script):
 
     """Force calculate_presentation() to be called on some set of Editions.
-
```

Comment:
There are a number of places where blank lines were removed when they shouldn't have been -- probably done automatically by Atom.